src/java_bytecode/CMakeLists.txt

```diff
@@ -1,6 +1,26 @@
+get_filename_component(JAVA_CORE_LIBRARY_PATH "library" REALPATH BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+get_filename_component(CORE_MODELS_JAR "library/core-models.jar" REALPATH BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Commit can be squashed into https://github.com/cesaro/cbmc/pull/5/commits/068c730d5164434a1b3c5894d57b04e361cef962